### DISCUSSION TITLE
warning 预计一次更新(......行)超过 5000 行.

### DISCUSSION THREAD

**Discussion by dearsiang** [hhyo/Archery#2470] [Q&A] [answered]
Hi 

請問在執行sql 審核時，在審核執行狀態，出現Warning , 且審核執行信息，出現预计一次更新(.....行)超过 5000 行.

這個5000行數，可否藉由archery 介面調整？

**Comment by LeoQuote** [comment]
据我所知所有的审核信息都是 goinception 提供的，所以是需要调整goinception 的配置才行

**Reply by dearsiang** [reply]
這邊有配置--real_row_count = "true"  , 在archery 容器下的/opt/archery/sql/engines/goinception.py, 這個檔案，所以影響行數沒有問題的，但這邊想調整這個Warning 出現的5000行數，是否可調整到10000行呢？

**Reply by nick2wang** [reply, marked as answer]
goinception的审核项可参考：https://hanchuanchuan.github.io/goInception/zh/options.html#variables列表
![image](https://github.com/hhyo/Archery/assets/33473924/48e59fab-d3ce-4dab-8633-ef1f6e6252bd)
